src/pybind/mgr/dashboard/frontend/package-lock.json

```diff
@@ -499,7 +499,8 @@
             "ansi-regex": {
               "version": "2.1.1",
               "bundled": true,
-              "dev": true
+              "dev": true,
```

Comment:
Please remove ``package-lock.json`` from this PR. I think this has been added by accident.